### PR TITLE
Add Candle (CNDL) Chain

### PR DESCRIPTION
**Pull request by samisbakedham**

### CODE REVIEW

File: _data/chains/eip155-534.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "name": "Candle Mainnet",
+    "chain": "Candle",
+    "rpc": [
+        "https://candle-rpc.com/",
+        "https://rpc.cndlchain.com",
```
Comment (ligi) [inline _data/chains/eip155-534.json:6]:
```suggestion
        "https://rpc.cndlchain.com"
```

**Review by ligi — CHANGES_REQUESTED** [review]
Please remove the comma to make it valid json

**Comment by samisbakedham** [comment]
> Please remove the comma to make it valid json

Done thanks!

**Comment by ligi** [comment]
I cannot get the IPFS CID - where did you pin it?

**Comment by samisbakedham** [comment]
> I cannot get the IPFS CID - where did you pin it?

Piñata

**Review by ligi — CHANGES_REQUESTED** [review]
oh it took a bit longer to load. But the size seems to be off:

```
✓ ~/tmp identify QmRVVjhK4rBgRNXNX46YJG7epbDYfGXL3ERomrBkuG8nAu         11:37:14
QmRVVjhK4rBgRNXNX46YJG7epbDYfGXL3ERomrBkuG8nAu PNG 6385x5665 oh it took a bit longer to load. But the size seems to be off:

```
✓ ~/tmp identify QmRVVjhK4rBgRNXNX46YJG7epbDYfGXL3ERomrBkuG8nAu         11:37:14
QmRVVjhK4rBgRNXNX46YJG7epbDYfGXL3ERomrBkuG8nAu PNG 6385x5665 6385x56656385x5665+0+0 8-bit sRGB 852012B 0.000u 0:00.000

```

 6385x5665 is quite big and should not be the only icon - please provide a smaller one

**Comment by samisbakedham** [comment]
> oh it took a bit longer to load. But the size seems to be off:
> 
> 
> 
> ```
> 
> ✓ ~/tmp identify QmRVVjhK4rBgRNXNX46YJG7epbDYfGXL3ERomrBkuG8nAu         11:37:14
> 
> QmRVVjhK4rBgRNXNX46YJG7epbDYfGXL3ERomrBkuG8nAu PNG 6385x5665 6385x56656385x5665+0+0 8-bit sRGB 852012B 0.000u 0:00.000
> 
> ```
> 
> 
> 
>  6385x5665 is quite big and should not be the only icon - please provide a smaller one 

Should I change it to a smaller size or add another icon? Thanks by the way

**Comment by samisbakedham** [comment]
Fixed

**Comment by samisbakedham** [comment]
> oh it took a bit longer to load. But the size seems to be off:
> 
> ```
> ✓ ~/tmp identify QmRVVjhK4rBgRNXNX46YJG7epbDYfGXL3ERomrBkuG8nAu         11:37:14
> QmRVVjhK4rBgRNXNX46YJG7epbDYfGXL3ERomrBkuG8nAu PNG 6385x5665 6385x56656385x5665+0+0 8-bit sRGB 852012B 0.000u 0:00.000
> ```
> 
> 6385x5665 is quite big and should not be the only icon - please provide a smaller one

All good to merge?

**Comment by samisbakedham** [comment]
@ligi

**Comment by ligi** [comment]
did you pin the smaller size? Trying to get this CID since 30min - no luck

**Comment by samisbakedham** [comment]
> did you pin the smaller size? Trying to get this CID since 30min - no luck

Shoot, I'll check! Apologies for using your time in a wasteful manner, I'll get ON this ASAP

**Comment by samisbakedham** [comment]
> did you pin the smaller size? Trying to get this CID since 30min - no luck

Try now!

**Comment by samisbakedham** [comment]
Yup, tested again and it shows up, should be okay now. Check the link [here](https://cloudflare-ipfs.com/ipfs/Qmbe3ChumXNRfHcLsNBY2APRrGxkFWP68Nb35MaKMRfPye)

**Comment by samisbakedham** [comment]
Running `Ipfs get Qmbe3ChumXNRfHcLsNBY2APRrGxkFWP68Nb35MaKMRfPye` seems to work as well I think. Attached image shows the result, if you'd like to try again @ligi ? image

**Comment by samisbakedham** [comment]
Any updates @ligi ?

**Comment by ligi** [comment]
I could get the image now - but your explorer is not accessible

**Comment by samisbakedham** [comment]
> I could get the image now - but your explorer is not accessible

Routine maintenance, check now

**Review by ligi — CHANGES_REQUESTED** [review]
Your explorer does not seem to conform to EIP3091 - please change to `none`

**Comment by samisbakedham** [comment]
> Your explorer does not seem to conform to EIP3091 - please change to `none`

Fixed

**Comment by ligi** [comment]
No I do not mean remove the explorer ( and leave an empty object) - I mean set the standard to `none`

**Comment by samisbakedham** [comment]
> No I do not mean remove the explorer ( and leave an empty object) - I mean set the standard to `none`



> No I do not mean remove the explorer ( and leave an empty object) - I mean set the standard to `none`

Done!

**Comment by samisbakedham** [comment]
@ligi Apologies but in order to build an adapter for DefiLlama we need this to be merged, preferably sooner rather than later. I understand you're extremely busy, is there any way we can give this the green light? I think it's good to go now. Thank you

**Review by ligi — APPROVED** [review]

**Comment by samisbakedham** [comment]
Just saw it was approved - thank you! Will it be merged now?

**Comment by 0xngmi** [comment]
having your chain listed here is NOT a requirement to build an adapter for defillama. Could you point me to the source where you got that idea @samisbakedham ?

**Comment by samisbakedham** [comment]
> having your chain listed here is NOT a requirement to build an adapter for defillama. Could you point me to the source where you got that idea @samisbakedham ?

https://github.com/DefiLlama/chainlist/pull/19

**Pull request merged**